directorynameproject2/directorynameproject2/Questionaire.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Questionaire
+{
+    public class Questionaire //the properties are here. these are the blueprint of your methods. cookie cutter, not the cookie
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public decimal FavoriteNumber { get; set; }
+        public decimal NumberOfPets { get; set; }
+        //public List<string> NameOfPets { get; set; } = new List<string>();//this is now a list. was a string <failed attempt>
+        public string NameOfPets { get; set;  }
```

Comment:
I like that you tried to go with a List<string>, but then decided to fall back to something simpler so as to get it working. Keep this failed attempt in mind and come back later to try again and make it work 👍